File: 001-100/001/multiplesOf3or5.cpp
# include <iostream>
using namespace std;

// Function to make calculation
int multiplesOf3or5(int n){
  int sum = 0;
  for(int i = 0; i < n; i++){
    if(i % 3 == 0 || i % 5 == 0){ sum += i; }
  }
  return sum;
}

int main(){
  // Get user input
  int n;
  cout << "Enter an integer n: ";
  cin >> n;
  int multiplesSum = multiplesOf3or5(n);
  // Print result
  cout << "Sum of multiples of 3 or 5 below " << n << endl;
  cout << multiplesSum;
  return 0;
}

File: 001-100/001/multiplesOf3or5.php
<?php
  // Calculation function
  function multiplesOf3or5($n){
    $sum = 0;
    for($i = 0; $i < $n; $i++){
      if($i % 3 == 0 || $i % 5 == 0){
        $sum += $i;
      }
    }
    return $sum;
  }

  // Get User input
  $prompt="Enter an integer n: ";
  $n = readline($prompt);
  $multiplesSum = multiplesOf3or5($n);
  // Return Result
  echo "\nSum of multiples of 3 or 5 below $n\n";
  echo "$multiplesSum\n";
?>
